apps/meteor/app/livechat/server/lib/Contacts.ts

```diff
@@ -344,3 +344,20 @@ export async function validateContactManager(contactManagerUserId: string) {
 		throw new Error('error-contact-manager-not-found');
 	}
 }
+
+export async function unverifyContactChannel(
+	contact: Pick<ILivechatContact, '_id' | 'channels'>,
+	channelName: string,
+	visitorId: string,
+): Promise<ILivechatContact> {
+	const channelToUnverify = contact.channels?.find((channel) => channel.name === channelName && channel.visitorId === visitorId);
```

Comment:
Where are using this variable? This seems to not being used anywhere...

Reply:
we use it in line 359:
```ts
channelToUnverify.verified = false;
```
since the .find method returns a reference to the member of the array we can use to mutate it without having to copy stuff

Reply:
we are doing the same thing over here: https://github.com/RocketChat/Rocket.Chat/pull/33491/files#diff-e4b60a5e991fbca9dd0894a0aa751cd11dc4d47948da1653c524989b56b2aa2fR376

Reply:
I got it, but `channelToUnverify` is not being used, is it?